bench/wtperf/wtperf_run_py/wtperf_run.py

```diff
@@ -48,10 +48,11 @@
 test_stats_file = 'test.stat'
 
 
-def create_test_home_path(home: str, test_run: int, operation: str):
+def create_test_home_path(home: str, test_run: int, operations: list = None):
     home_path = "{}_{}".format(home, test_run)
-    if operation is not None: 
-        home_path += "_{}".format(operation)
+    if operations:
+        # Use the first operation name as part of the home path
+        home_path += "_{}".format(operations[0])
```

Comment:
There's a risk here of someone defining an arg_file as
```
{
      "argument": "...",
      "operations": [ "load", "read" ]
    },
    {
      "argument": "...",
      "operations": [ "load", "update" ]
    },
```
causing two tests to have the same `test_home_path`. 
I think it's an improper use case but raising for discussion.

Reply:
I agree, however, the assumption is that the arg_file will have valid arguments and will not consist of the same operations (for instance, "load", in this case above should not be repeated). 
I do think that there should be a validation function that checks the same.

Reply:
I agree, I think it would be good to defend against such errors. At the moment, we only have two argument files, so it's easy to check manually, but I've created ticket [WT-8326](https://jira.mongodb.org/browse/WT-8326) so that we can add code to defend against such an error.